Gemfile

```diff
@@ -0,0 +1,34 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "3.2.2"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.0.6"
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem "sprockets-rails", "~> 3.4"
```

Comment:
We don't have `sprocket-rails` in our other apps, do we need it here?

Reply:
No, I think we can [get rid of it as we have vite, but equally we can keep it if it's useful.](https://github.com/ElMassimo/vite_ruby/pull/165) I think it's better to start with it and remove it as it's tied to the assets:precompile stuff when rails is built.

I'll stick a PR with the asset build after this.